### DISCUSSION TITLE
Plans for null safety migration

### DISCUSSION THREAD

**Discussion by kiranvivo** [smaho-engineering/esptouch_flutter#21] [General]
Hi,

Just interested to know if null safety is WIP or no plans for it in near future?

**Comment by vincevargadev** [comment]
I'm planning to do it this week

**Reply by kiranvivo** [reply]
Great. Thank you.

**Comment by vincevargadev** [comment]
Could you please check and provide some feedback? Check version 1.0.0

https://pub.dev/packages/esptouch_flutter/changelog

**Reply by kiranvivo** [reply]
That was fast. Sure, I will check it out. 

Btw, I am building a ESP based project and just got started. What are your thoughts for provisioning ? ESP Touch is reliable enough or having a backup softAP provisioning is recommended ?

**Comment by kiranvivo** [comment]
Hi again,

I was able to the get the app up and running. Thank:) 

I have noticed the following warning:

The plugin `esptouch_flutter` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding.
Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding:
https://flutter.dev/go/android-plugin-migration.

Any comments on the same ?

**Reply by vincevargadev** [reply]
You can follow the issue here https://github.com/smaho-engineering/esptouch_flutter/issues/19

With that said, I don't know if/when I'll have the time for the migration, so community contributions are definitely welcome!
